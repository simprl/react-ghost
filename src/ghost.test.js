import { create } from 'react-test-renderer'
import { ghost, ghosts } from './index';

const Ghost = () => {
    return null;
};

describe('init', () => {
    test('create app actor', () => {
        const mainGhost = create(ghosts(ghost(Ghost)));
        expect(mainGhost.toJSON()).toMatchSnapshot();
    });
});
