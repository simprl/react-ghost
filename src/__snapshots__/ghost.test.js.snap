// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`init create app actor 1`] = `null`;
